mhcflurry/class1_affinity_prediction/train_allele_specific_models_command.py

```diff
@@ -48,7 +48,18 @@
     type=int,
     help="Keras verbosity. Default: %(default)s",
     default=1)
-
+parser.add_argument(
+    "--use-kubeface",
+    type=bool,
+    help="Use Kubeface: %(default)s",
```

Comment:
Standard way to do bool args is to say `default=False, action="store_true"`. Then you just say `--use-kubeface` to turn on the flag